main.scss

```diff
@@ -50,6 +51,16 @@
 
 	.o-tooltip-close {
 		@include oTooltipClose;
+
+		.o-share__action & {
```

Comment:
It should. @gvonkoss and I had a chat about this but I'm not sure if this was left in by accident or there was a reason. This (ideally) would be something that was explained in the commit message (hint hint). Gabi when you're back can you give me a refresher on this?